utils/portalloc/src/data/dto.rs

```diff
@@ -37,6 +43,8 @@ impl Default for RootData {
 
 impl RootData {
     pub fn get_free_port_range(&mut self, range_size: u16) -> u16 {
+        self.reclaim(now_ts());
```

Comment:
This now will not work with `&self`, but possibly can be moved out too.

Reply:
I'm not sure what you mean here - can you clarify?